contracts/mocks/MockMarginCalculator.sol

```diff
@@ -206,7 +211,10 @@ contract MockMarginCalculator {
     }
 
     function _getUnderlyingPrice(address _otoken) internal view returns (uint256, bool) {
-        return (price[_otoken], isPriceFinalized[_otoken]);
+        OracleInterface oracle = OracleInterface(AddressBookInterface(addressBook).getOracle());
+        OtokenInterface otoken = OtokenInterface(_otoken);
+
+        return oracle.getExpiryPrice(otoken.underlyingAsset(), otoken.expiryTimestamp());
```

Comment:
Why using the oracle contract? 
This will break other tests.

Reply:
I think we should keep the mock contract simple, why adding oracle into the dependency graph

Reply:
It is is simple, It just call another Mock contract to get price.
I made that change because the exercise function use the Oracle mock, the the `_getUnderlyingPrice` was using a mapping to mock prices, so I needed to push price+finalized in two mocks for exercise to work.
Better to have both of them using same Oracle mock. Also this does not break any tests.